reference/reference-environment-variables.md

```diff
@@ -489,8 +489,11 @@ to be routed through the exit node.
 {{<table "table table- bordered" "text-align:center" >}}
 | <center>Name</center> | <center>Description</center>                                                                    | <center>Default value</center> | <center>Read Only</center> |
 | --------------------- | ----------------------------------------------------------------------------------------------- | ------------------------------ | -------------------------- |
+| CC_VPN_DNS_OVERRIDES  | Comma-separated list of DNS IP                                                                  |                                |                            |
 | VPN_ADDON_CRT         | Client certificate                                                                              |                                |                            |
+| VPN_ADDON_DEVTYPE     | Kernel virtual interface kind to use                                                            | tap                            |                            |
```

Comment:
@miton18 can you put the different values (tap, tun) ?

Reply:
@miton18 UP

Reply:
```suggestion
| VPN_ADDON_DEVTYPE     | Kernel virtual interface kind to use                                                            | tap                            |         tap, tun                 |
```